doc/whats_new.rst

```diff
@@ -15,26 +15,40 @@ Changelog
 New features
 ............
 
-   - The Gaussian Process module has been reimplemented and now offers classification 
-     and regression estimators through :class:`gaussian_process.GaussianProcessClassifier` 
-     and  :class:`gaussian_process.GaussianProcessRegressor`. Among other things, the new 
-     implementation supports kernel engineering, gradient-based hyperparameter optimization or 
-     sampling of functions from GP prior and GP posterior. Extensive documentation and 
+   - The Gaussian Process module has been reimplemented and now offers classification
+     and regression estimators through :class:`gaussian_process.GaussianProcessClassifier`
+     and  :class:`gaussian_process.GaussianProcessRegressor`. Among other things, the new
+     implementation supports kernel engineering, gradient-based hyperparameter optimization or
+     sampling of functions from GP prior and GP posterior. Extensive documentation and
      examples are provided. By `Jan Hendrik Metzen`_.
- 
+
 Enhancements
 ............
- 
+
 Bug fixes
 .........
-    - Fixed bug in :func:`manifold.spectral_embedding` where diagonal of unnormalized 
+
+    - :class:`RandomizedPCA` default number of `iterated_power` is 2 instead of 3.
+      This is a speed up with a minor precision decrease. By `Giorgio Patrini`_.
+
+    - :func:`randomized_svd` performs 2 power iterations by default, instead or 0.
+      In practice this is often enough for obtaining a good approximation of the
+      true eigenvalues/vectors in the presence of noise. By `Giorgio Patrini`_.
+
+    - :func:`randomized_range_finder` is more numerically stable when many
```

Comment:
can you please add a link to the PR to whatsnew? And ``versionadded`` hints for the changed parameters would be nice ;)